File: Modul 3/Soal_4/PRAK304-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
angka = int(input())

if angka == 0 :
    print("Nol")
elif angka > 99 :
    print("Anda Menginput Melebihi Limit Bilangan")
elif angka == 10 or angka >= 20 :
    print("Puluhan")
elif angka > 10 :
    print("Belasan")
elif angka > 0 :
    print("Satuan")

File: Modul 3/Soal_4/PRAK304-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>

int main(){
    int angka;

    scanf("%d", &angka);

    if(angka == 0){
        printf("Nol");
    }else if(angka > 99){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }else if(angka == 10 || angka >= 20){
        printf("Puluhan");
    }else if (angka > 10)
    {
        printf("Belasan");
    }else if (angka > 0){
        printf("Satuan");
    }
    

    return 0;
}